.github/workflows/on-code-change.yml

```diff
@@ -72,10 +72,11 @@ jobs:
           module_id: content-editor
           testrail_project: Content Editor
           tests_manifest: provisioning-manifest-build.yml
-          jahia_image: jahia/jahia-discovery-dev:8-SNAPSHOT
           should_use_build_artifacts: true
-          ci_startup_script: "ci.startup.sh"
+          jahia_cluster_enabled: false
           should_skip_testrail: true
+          jahia_image: jahia/jahia-ee-dev:8-SNAPSHOT
+          ci_startup_script: "ci.startup.sh"
```

Comment:
Not needed here as it's the same value that is provided by default here: https://github.com/Jahia/jahia-modules-action/blob/main/integration-tests/action.yml#L182 (and used here: https://github.com/Jahia/jahia-modules-action/blob/main/integration-tests/src/docker/start.ts#L21)